src/main.lua

```diff
@@ -1254,27 +1248,20 @@ addEventingHandler(
 	ActionMap.DecreaseDelegateStake,
 	utils.hasMatchingTag("Action", ActionMap.DecreaseDelegateStake),
 	function(msg)
-		local from = utils.formatAddress(msg.From)
-		local target = utils.formatAddress(msg.Tags.Target or msg.Tags.Address)
+		local target = msg.Tags.Target or msg.Tags.Address
 		local quantity = tonumber(msg.Tags.Quantity)
 		local instantWithdraw = msg.Tags.Instant and msg.Tags.Instant == "true" or false
 		local timestamp = tonumber(msg.Timestamp)
-		local messageId = msg.Id
 		msg.ioEvent:addField("Target-Formatted", target)
 		msg.ioEvent:addField("Quantity", quantity)
-		assert(utils.isValidAOAddress(target), "Invalid gateway address")
 		assert(
-			quantity
-				and tonumber(msg.Tags.Quantity) > constants.minimumWithdrawalAmount
-				and utils.isInteger(msg.Tags.Quantity),
+			quantity and utils.isInteger(quantity) and quantity > constants.minimumWithdrawalAmount,
 			"Invalid quantity. Must be integer greater than " .. constants.minimumWithdrawalAmount
 		)
 
-		local result = gar.decreaseDelegateStake(target, from, quantity, timestamp, messageId, instantWithdraw)
-
+		local result = gar.decreaseDelegateStake(target, msg.From, quantity, timestamp, msg.Id, instantWithdraw)
 		local decreaseDelegateStakeResult = {
 			gateway = result and result.gateway or {},
-			penaltyRate = result and result.penaltyRate or 0,
```

Comment:
Is this relevant to the PR?

Reply:
[20e7aa0](https://github.com/ar-io/ar-io-network-process/pull/183/commits/20e7aa08098a40ba6ca75d3f28c629d04c9d8c45)